### ISSUE TITLE
Wishbone marker selection

### ISSUE THREAD

**Issue by wy-18** [ManuSetty/wishbone#71]
Dear Dr. Setty,

I noticed that you chose 13 surface markers from the panel. I would like to know how do you select these 13 markers? Does the number of markers matter for the algorithm?

Bests,

Cathy

**Comment by ManuSetty** [comment]
The markers were chosen based on the prior biological knowledge. The number of markers should not matter to the algorithm.

**Comment by wy-18** [comment]
Okay, so for wishbone paper, you select 13 from 31 markers. Is this because 13 is enough? Also, we noticed that there are two panels in Wishbone paper and would like to learn how do you combine these two panels?

Thank you

**Comment by ManuSetty** [comment]
Yes - we selected the 13 markers out of 31. We did not combine the two panels and ran Wishbone separately on the two sets of samples
